src/components/app/App.jsx

```diff
@@ -33,7 +33,7 @@ const queryClient = new QueryClient({
       // relying on the cached data until the `staleTime` window has exceeded. This may be modified
       // per-query, as needed, if certain queries expect to be more up-to-date than others. Allows
       // `useQuery` to be used as a state manager.
-      staleTime: 1000 * 20,
+      staleTime: 1000 * 6,
```

Comment:
[inform] This was temporary while looking at search results re-rendering this morning. Will be changed back to 20s.